### PR TITLE
Action to comment on a Jira issue

### PR DESCRIPTION
**Pull request by psychopenguin**

### CODE REVIEW

File: packs/jira/actions/comment_issue.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: comment_issue
+runner_type: run-python
+description: Comment on a JIRA issue / ticket.
+enabled: true
+entry_point: comment_issue.py
+parameters:
+  text:
```
Comment (manasdk) [inline packs/jira/actions/comment_issue.yaml]: suggest `comment` or `comment_text`.

File: packs/jira/actions/comment_issue.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    type: string
+    description: the comment itself.
+    required: true
+  issue_key:
+    type: string
+    description: Issue key (e.g. PROJECT-1000).
+    required: true
```
Comment (manasdk) [inline packs/jira/actions/comment_issue.yaml]: New line please :D

**Comment by psychopenguin** [comment]
@manasdk just pushed your sugestions. :)

**Comment by manasdk** [comment]
+1

Thanks!

**Pull request merged**